features/image-transformations/resize-crop-and-other-transformations.md

```diff
@@ -875,3 +875,75 @@ By specifying this parameter in the file URL, ImageKit sets the `content-disposi
 Example - [https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?ik-attachment=true](https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?ik-attachment=true)
 
 ![](https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?ik-attachment=true)
+
+### Page - (pg)
+Delivers specified page, frame, or layer of a multi-page/frame/layer file, such as a PDF, animated image like GIF, or PSD. It needs to be used alongside the ik-thumbnail.jpg URL component.
```

Comment:
Can add a link to the `ik-thumbnail` section/page

Reply:
Not adding, it might be confusing as it is used in completely different context there.